ci/pipeline.yml

```
@@ -0,0 +1,27 @@
+include:
+  - remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
+
+stages:
+  - build
+  - test
+
+variables:
+  PERSIST_IMAGE_NAME: $CSCS_REGISTRY_PATH/quda:$CI_COMMIT_SHORT_SHA
```

can you include this next time you touch it? this will allow me to access it outside of ci

```suggestion
  PERSIST_IMAGE_NAME: $CSCS_REGISTRY_PATH/quda/public/build:$CI_COMMIT_SHORT_SHA
```